tests/scripts/fetch_target_logs.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
```

Why not `set -eux`? `-eu` is safeguarding against many hidden errors, failing early rather than late.

---

I think we don't want to fail the CI run because of the log collection error. That is why we use only `-x`.

---

Yes, true.